src/org/labkey/test/components/ui/grids/GridBar.java

```diff
@@ -433,6 +442,12 @@ public String getSearchExpression()
         return elementCache().searchBox.get();
     }
 
+    public GridFilterModal getFilterDialog()
+    {
+        clickButton("Filters");
+        return new GridFilterModal(getDriver(), _queryGrid);
+    }
+
```

Comment:
`get` methods generally shouldn't change the page state. Maybe `openFilterDialog` instead.

Reply:
Changed.